charts/argo-cd/values.yaml

```diff
@@ -1694,6 +1702,23 @@ repoServer:
 
 ## Argo Configs
 configs:
+  # Provide cache expirations
+  cache:
+    # -- Default cache expiration
+    default: 24h
```

Comment:
What do you want to do with `cache` and `repoServer` config? I've seen that this is cross-cutting concern where same value repeats for multiple components and I'm not sure if we want to somehow simplify this or we want to keep it dry and somehow merge / template 1 section into params as well.